src/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.wrap {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  width: 33.33%;
```

Comment:
this is a flex container. let's learn flex and use `flex-basis`. also 33.33% is not quite what we want, because 33.33 * 3 is 99.99, but not 100. let's use `calc(100% / 3)`